packages/beagle-react/src/components/BeagleDynamicLists/DynamicListCore/index.tsx

```diff
@@ -17,13 +17,16 @@
 import React, { FC, useEffect, useRef, Children } from 'react'
 import { BeagleUIElement } from '@zup-it/beagle-web'
 import { Tree, logger } from '@zup-it/beagle-web'
-import withTheme from '../utils/withTheme'
-import { buildAccessibility } from '../../../../common/utils/accessibility'
+import withTheme from '../../utils/withTheme'
+import { BeagleGridViewInterface, BeagleListViewInterface, DynamicListCoreInterface } from '../../../../../common/models'
+import { buildAccessibility } from '../../../../../common/utils/accessibility'
 import useScroll from './scroll'
-import { StyledListView } from './styled'
-import { BeagleListViewInterface } from './types'
+import { StyledDynamicViewsInterface } from './styled'
 
-const BeagleListView: FC<BeagleListViewInterface> = ({
+
+interface DynamicViewInterface extends BeagleListViewInterface, BeagleGridViewInterface { }
+
+const DynamicListCoreComponent: FC<DynamicViewInterface> = ({
   direction = 'VERTICAL',
```

Comment:
Based on the suggestion of using String Enums (down on this PR), I suggest:

```suggestion
  direction = ListDirection.Vertical,
```